paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -223,19 +246,22 @@ def deploy_group_is_set_to_notify(self):
     def post(self, **kwargs):
         if self.deploy_group_is_set_to_notify():
             self.sc.post(**kwargs)
+        else:
+            log.debug(f"{self.deploy_group} isn't set to notify slack")
 
     def notify_after_mark(self, ret):
         if ret == 0:
-            message = (
-                f"*{self.service}* - Marked *{self.commit}* for deployment on *{self.deploy_group}*. "
-                "Will start deploying soon!"
+            mes = (
+                f"*{self.service}* - Marked *{self.commit}* for deployment on *{self.deploy_group}*.\n"
+                "Will start deploying soon!\n"
+                f"{self.authors}"
             )
-            self.post(channels=self.channels, message=message)
+            self.post(channels=self.channels, message=mes)
             if self.old_commit is not None and self.commit != self.old_commit:
                 mes = (
                     "Roll it back at any time with:\n"
-                    f"```paasta rollback --service {self.service} --deploy-group {self.deploy_group} "
-                    f"--commit {self.old_commit}```"
+                    f"`paasta rollback --service {self.service} --deploy-group {self.deploy_group} "
+                    f"--commit {self.old_commit}`"
```

Comment:
wonder if this message will be confusing for y-m where that's _not_ the right way to initiate a rollback